keyboards/hineybush/hbcp/config.h

```diff
@@ -27,6 +27,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT         hbcp
 #define DESCRIPTION     compact 1800 layout custom keyboard
 
+/* needed because PCBA used the wrong diode part */
+#define EITHERWAY 3 /* Compensation for diode direction dissonance */
+
```

Comment:
Technically, this isn't needed right now, as the custom matrix doesn't check for it.

Reply:
should I keep it for now, so it'll work with future master-merged code?

Reply:
DIODE_DIRECTION would be the important bit (plus having this here before names and values are decided will potentially cause additional issues)
```suggestion
```